stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/ChannelActionsDialogViewStyle.kt

```diff
@@ -0,0 +1,186 @@
+                        context.getColorCompat(R.color.stream_ui_text_color_primary)
+                    )
+                    .font(
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsMemberNamesTextFontAssets,
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsMemberNamesTextFont
+                    )
+                    .style(
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsMemberNamesTextStyle,
+                        Typeface.BOLD
+                    )
+                    .build()
+
+                val memberInfoTextStyle = TextStyle.Builder(a)
+                    .size(
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsMemberInfoTextSize,
+                        context.getDimension(R.dimen.stream_ui_text_small)
+                    )
+                    .color(
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsMemberInfoTextColor,
+                        context.getColorCompat(R.color.stream_ui_text_color_secondary)
+                    )
+                    .font(
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsMemberInfoTextFontAssets,
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsMemberInfoTextFont
+                    )
+                    .style(
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsMemberInfoTextStyle,
+                        Typeface.NORMAL
+                    )
+                    .build()
+
+                val itemTextStyle = TextStyle.Builder(a)
+                    .size(
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsItemTextSize,
+                        context.getDimension(R.dimen.stream_ui_text_medium)
+                    )
+                    .color(
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsItemTextColor,
+                        context.getColorCompat(R.color.stream_ui_text_color_primary)
+                    )
+                    .font(
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsItemTextFontAssets,
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsItemTextFont
+                    )
+                    .style(
+                        R.styleable.ChannelActionsDialog_streamUiChannelActionsItemTextStyle,
+                        Typeface.BOLD
+                    )
+                    .build()
+
+                val viewInfoIcon = a.getDrawable(R.styleable.ChannelActionsDialog_streamUiChannelActionsViewInfoIcon)
+                    ?: context.getDrawableCompat(R.drawable.stream_ui_ic_single_user)!!
+
+                val viewInfoEnabled = a.getBoolean(
+                    R.styleable.ChannelActionsDialog_streamUiChannelActionsViewInfoEnabled,
+                    true
```

Comment:
I think it should be disabled by default because we don't provide default action for this button